lib/color_spinor_field.cpp

```diff
@@ -454,7 +454,7 @@ namespace quda
 
       if (reorder_location() == QUDA_CPU_FIELD_LOCATION) { // reorder on the host
         void *buffer = pool_pinned_malloc(bytes);
-        qudaMemcpy(buffer, v.data(), bytes, qudaMemcpyDefault);
+        qudaMemcpy(buffer, src.v.data(), bytes, qudaMemcpyDefault);
```

Comment:
I observed that reordering on the CPU always results in vectors that are zero everywhere. I think there is a typo, since this fixed reordering for me.